docs/nuvlabox/nuvlabox-engine/installation.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Advanced Installation
+nav_order: 3
```

Comment:
this section was removed. Why put it back in?

Reply:
probably a merge error. My bad. Fixed now.